features/dd-sdk-android-rum/src/test/kotlin/com/datadog/android/rum/internal/metric/RumSessionEndedMetricTest.kt

```diff
@@ -386,10 +436,89 @@ class RumSessionEndedMetricTest {
         assertThat(sdkErrorCount[SessionEndedMetric.SDK_ERRORS_COUNT_BY_KIND_KEY]).isEqualTo(expectedErrorKind)
     }
 
+    @Test
+    fun `M have correct 'no_view_events_count' W toMetricAttributes()`(
+        @BoolForgery fakeBackgroundEventTracking: Boolean,
+        @IntForgery(min = 1, max = 5) randomActionCount: Int,
+        @IntForgery(min = 1, max = 5) randomResourceCount: Int,
+        @IntForgery(min = 1, max = 5) randomErrorCount: Int,
+        @IntForgery(min = 1, max = 5) randomLongTaskCount: Int
```

Comment:
by setting `min = 1` we are missing than the case in the code when the particular type was never seen. Maybe we should set `min = 0`? This will cover fallback `?: 0` in the code.

Reply:
Indeed, fixed